ddp-workspace/projects/ddp-pancan/src/app/components/anchors-page/anchors-page.component.html

```diff
@@ -4,25 +4,44 @@ <h2 class="title static-page-title">{{source + '.Title' | translate}}</h2>
     <nav class="anchors">
         <ul>
             <li *ngFor="let section of (source + '.Sections') | translate; let i = index;" class="anchor">
-                <a [routerLink]="['/', route]" [fragment]="anchorLabel + i">{{section.Title}}</a>
+                <a [routerLink]="['/', route]" [fragment]="anchorLabel + i"
+                   (click)="scrollToAnchor(anchorLabel + i)">{{section.Title}}
+                </a>
             </li>
         </ul>
     </nav>
 
     <section [id]="anchorLabel + i"
              *ngFor="let section of (source + '.Sections') | translate; let i = index;"
              class="section">
-        <div class="section__content" [class.column-direction]="section.PeopleList?.length">
+        <div class="section__content"
+             [class.column-direction]="section.PeopleList?.length">
             <h3 class="section__title static-page-title">{{section.Title}}</h3>
 
-            <div class="section__description" *ngIf="section.Paragraphs?.length">
-                <ng-container *ngFor="let paragraph of section.Paragraphs">
-                    <p class="section__text static-page-paragraph-text" [innerHTML]="paragraph" routeTransformer></p>
-                </ng-container>
+            <div class="section__description" *ngIf="section.Paragraphs?.length || section.Examples">
+                <div *ngIf="section.Paragraphs?.length">
```

Comment:
#minor do we need an additional div here or we  can use ng-container?

Reply:
Sure, it can be ng-container.
Thanks, fixed.